packages/compiler/core/checker.ts

```diff
@@ -1931,6 +1947,11 @@ export function createChecker(program: Program): Checker {
       if (binding) return binding.flags & SymbolFlags.DuplicateUsing ? undefined : binding;
     }
 
+    // Until we have an `unit` type for `null`
+    if (node.sv === "null") {
```

Comment:
I know these are meant to be temporary, but it still feels off that we need special-casing around null in the interim. Would it be possible to put a symbol into the symbol table for it. It seems like maybe this could be done in initializeCadlIntrinsics.

Also, I think we'll lose `null` completion without another special-case in resolveCompletions.

Reply:
Updated to connect the symbol to the globalNamespace exports and the type instead of having hte special if checks.
Also checked that complete works.